src/index.html

```diff
@@ -6,9 +6,66 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+        class="logo__img"
+        src="images/logo.png"
+        alt="MoYo_logo"
```

Comment:
```suggestion
          class="logo__img"
          src="images/logo.png"
          alt="MoYo_logo"
```
Consider fixing indentations according to the code style guide